### PR TITLE
gh-1472 clarify fiber.yield description + some translation fixes

### PR DESCRIPTION
**Pull request by Onvember**

### CODE REVIEW

File: locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po [hunk 1]
```diff
@@ -736,22 +736,30 @@ msgstr ""
 "экземпляра Tarantool'а называется 'interactive', а новые файберы, "
 "созданные с помощью :ref:`fiber.create <fiber-create>`, называются 'lua'."
 " Переименование файберов позволяет легче различать их при использовании "
-":ref:`fiber.info <fiber-info>`."
+":ref:`fiber.info <fiber-info>`. Максимум 32 символа."
 
 msgid "the new name of the fiber."
 msgstr "новое имя файбера."
 
+#, fuzzy
```
Comment (lenkis) [inline locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po:744]: if you validated the translation, please remove the fuzzy flag here

File: locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po [hunk 2]
```diff
@@ -1294,6 +1293,45 @@ msgstr ""
...
+"sleep>`), может упасть (выдать исключение)."
+
+msgid ""
+"For example, this function has a loop which repeats until :ref:`cancel() "
+"<fiber_object-cancel>` happens. The last thing that it will print is "
+"'before yield', which demonstrates that ``yield()`` failed, the loop did "
+"not continue until :ref:`testcancel() <fiber-testcancel>` failed."
+msgstr ""
+"Например, в этой функции есть цикл, который повторяется до тех пор, пока "
+"не произойдет :ref:`cancel() <fiber_object-cancel>`.Последнее, что она "
```
Comment (lenkis) [inline locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po:1315]:
```suggestion
"не произойдет :ref:`cancel() <fiber_object-cancel>`. Последнее, что она "
```

**Review by lenkis — CHANGES_REQUESTED** [review]

File: locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po (hunk 1 above)
Comment (Onvember) [reply locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po]: fixed in 9f63d45

File: locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po (hunk 2 above)
Comment (Onvember) [reply locale/ru/LC_MESSAGES/reference/reference_lua/fiber.po]: fixed in 9f63d45

**Pull request merged**